math/finding_number_of_Digits_in_a_Number.cpp

```diff
@@ -0,0 +1,33 @@
+/**
+ * @author amino19
+ * @file
+ *
+ * \brief Finding number of Digits in a Number
+ * \details Basically, its a very basic math of
+ *  finding number of digits in a number (input),
+ *  For more details, refer Algorithms-Explanation
+ */
+
+
+/* Log based C++ program to count number of
+*  digits in a number/integer */
+
+#include <bits/stdc++.h>
```

Comment:
```suggestion
#include <iostream>
```

Please read [CONTRIBUTING.md](https://github.com/TheAlgorithms/C-Plus-Plus/blob/master/CONTRIBUTING.md)
You cannot use `bits/stdc++.h`.

Reply:
Okay, I forgot to read this; thanks for your suggestion, cpyberry! 😄

Reply:
I am happy that I can help you :)